src/core/utils.ts

```diff
@@ -118,10 +118,73 @@ export function getSeriesMarkerType(series?: Highcharts.Series): ChartSeriesMark
 export function getSeriesColor(series?: Highcharts.Series): string {
   return typeof series?.color === "string" ? series.color : "black";
 }
+
+/**
+ * Extracts the color value from a solid gauge yAxis stops configuration.
+ * @param yAxis - The Highcharts yAxis object
+ * @returns The color string from the first stop, or "black" as fallback
+ */
+export function getSolidGaugeColor(yAxis?: Highcharts.Axis): string {
+  if (!yAxis) {
+    return "black";
+  }
+
+  try {
+    const yAxisOptions = yAxis as unknown as Highcharts.YAxisOptions;
```

Comment:
Why is conversion needed? I think the HC types usually provide `.options` or `.userOptions` to access the configuration.